src/Components/WebAssembly/WebAssembly/src/Hosting/WebAssemblyHostBuilder.cs

```diff
@@ -162,6 +163,18 @@ public void ConfigureContainer<TBuilder>(IServiceProviderFactory<TBuilder> facto
             };
         }
 
+        /// <summary>
+        /// Adds services to the container.
+        /// </summary>
+        /// <param name="configureService">The delegate for configuring the <see cref="IConfigurationBuilder"/> that will be used
+        /// to construct the <see cref="IConfiguration"/> for the host.</param>
+        /// <returns>The same instance of the <see cref="WebAssemblyHostBuilder"/> for chaining.</returns>
+        public WebAssemblyHostBuilder ConfigureServices(Action<IServiceCollection> configureService)
```

Comment:
I'm not super sure we need this indirection. If you have access to the builder, you should be able to just add services to it.